lcls_naming_tool/tests/test_cli.py

```diff
@@ -0,0 +1,86 @@
+# Unit tests for the LCLS Naming Tool script
+
+import pytest
+import json
+from pathlib import Path
+from lcls_naming_tool import display_version, load_taxons, functional_component_is_valid, fungible_is_valid, constituent_component_is_valid, increment_is_valid, starts_alphanumeric, validate
+
+
+@pytest.fixture
+def valid_name():
+    return ['MR2K4', 'KBO', 'PIP', '01', 'PUMPSIZE']
```

Comment:
It appears there are functional/fungible components mixed in here. I would argue that these should be separate fixtures, so we can test the cases more completely